pkgs/servers/matrix-synapse/plugins/rest-password-provider.nix

```diff
@@ -0,0 +1,13 @@
+{ buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "matrix-synapse-rest-password-provider";
+  version = "unstable-2020-08-02";
+
+  src = fetchFromGitHub {
+    owner = "ma1uta";
+    repo = pname;
+    rev = "c782c84aeab1872e73b6c29aadb99d3852e26bbd";
+    sha256 = "0v9900d6919yq05s08ihrc8jclpaixfm8mwi60vhxalmddgwlhjy";
+  };
```

Comment:
Missing meta

Reply:
no.

Reply:
sorry. Maybe? The other plugins don't have meta information either. So I assumed it's not supposed to be there

Reply:
As far as I know this is a package so it requires a meta section.

Reply:
@abbradar was there any particular reason we didn't add meta information to the synapse plugins?